mycore-csl/src/main/resources/components/csl/config/mycore.properties

```diff
@@ -0,0 +1,14 @@
+# MCR.ContentTransformer.mods2bibmods.Class=org.mycore.common.content.transformer.MCRXSLTransformer
+# MCR.ContentTransformer.mods2bibmods.Stylesheet=xsl/mycoreobject-mods.xsl,xsl/mods2bibmods.xsl
+# MCR.ContentTransformer.mods2bibmods.MIMEType=text/xml
+# MCR.ContentTransformer.mods2bibmods.FileExtension=xml
+
+# MCR.ContentTransformer.mods2bibtex.Class=org.mycore.common.content.transformer.MCRBibUtilsTransformer
+# local.property: system command for xml2bib
+# MCR.ContentTransformer.mods2bibtex.Command=%UBO.BibUtils.Base%xml2bib -b -w -o UTF8
```

Comment:
Do not use properties that are not defined elsewhere

Reply:
How about removing properties from mycore-mods and add a comment:
#requires runtime dependency mycore-mods and including the last 3 lines as active properties